### DISCUSSION TITLE
Welcome to Atropos Discussions!

### DISCUSSION THREAD

**Discussion by nolimits4web** [nolimits4web/atropos#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sami-20s** [comment]
hello My name is sami I'm a Front End Developer 
I'm useing the atropos in my react app and when i start my project he is tell me that :
module not found: Can't resolve 
what can i do ?

**Reply by Iipal** [reply]
++++
The message i see right now is: `Module not found: Can't resolve 'atropos/react' in 'fullpath/to/component'`.

Importing it like this:
```ts
import Atropos from "atropos/react";
```

This also not works:
```ts
import { Atropos } from "atropos/react";
```

Also, I use TypeScript.
